scripts/zkai_attention_kv_transition_receipt_probe.py

```
@@ -0,0 +1,365 @@
+            ),
+        },
+        "next_backend_step": (
+            "replace the source-backed transition receipt with a Stwo proof or source-backed proof receipt "
+            "that consumes the same public-instance fields"
+        ),
+    }
+    validate_payload(payload)
+    return payload
+
+
+def validate_payload(payload: Any) -> None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `payload`

(ANN401)